### PR TITLE
Codesandbox icon addition

### PR DESCRIPTION
**Pull request by LucasZapico**
**Issue:**
Codesandbox #1183

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/codesandbox.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+<svg viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg">
```
Comment (ericcornelissen) [inline icons/codesandbox.svg]: Could you remove `version="1.1"` and add `rule="img"`

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

File: icons/codesandbox.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+    <title>Codesandbox Icon</title>
```
Comment (ericcornelissen) [inline icons/codesandbox.svg]: Can you update this so it uses the correct capitalization for the brand name?

```suggestion
    <title>CodeSandbox Icon</title>
```

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simple-icons",
-  "version": "1.9.13",
+  "version": "1.9.18",
```
Comment (ericcornelissen) [inline package-lock.json]: Can you revert this change? Even if it should be updated, I'd prefer that not to be part of this PR 😄

File: _data/simple-icons.json [hunk 4]
```diff
@@ -385,6 +385,11 @@
             "hex": "000000",
             "source": "http://codepen.io"
         },
+        {
+          "title": "CodeSandbox",
+          "hex": "000000",
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Can you provide a source or reason for choosing this color?

File: icons/codesandbox.svg [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (ericcornelissen) [inline icons/codesandbox.svg]: We prefer the SVG to start no the first line 😄

File: _data/simple-icons.json [hunk 6]
```diff
@@ -385,6 +385,11 @@
             "hex": "000000",
             "source": "http://codepen.io"
         },
+        {
+          "title": "CodeSandbox",
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: I believe "Code**S**andbox" should come after "Code**r**wall" 😉

File: _data/simple-icons.json (hunk 4 above)
Comment (ericcornelissen) [reply _data/simple-icons.json]: In that regard, if I look at their social media ([Twitter](https://twitter.com/codesandbox) and [Spectrum](https://spectrum.chat/codesandbox)), I would expect the color to be white 🤔

Comment (LucasZapico) [reply _data/simple-icons.json]: Am I misunderstanding our format? Is the color in the JSON the fill color of the icon brand-wise? Or the background color of the icon displayed on [SimpleIcons](https://simpleicons.org/)?

Comment (ericcornelissen) [reply _data/simple-icons.json]: The color *should* be the "main" brand color and it is displayed on the website as the background color (although that is subject to change, see #980). The fill color of the SVG on the website is always black or white.  When you download the icon from the website it has no color specified.
